jdisc_core/src/test/java/com/yahoo/jdisc/core/ExportPackagesIT.java

```diff
@@ -52,6 +52,9 @@ public class ExportPackagesIT {
             "javax.activation.jar"
     ).map(f -> JAR_PATH + f).toList();
 
+    private static final List<String> newPackagesInJava21 = List.of("java.lang.foreign");
+    private static final List<String> removedPackagesInJava21 = List.of("com.sun.jarsigner");
```

Comment:
Changes since JDK17 or JDK20?

Reply:
`java.lang.foreign` also existed in 20 (don't know when it was added)
`com.sun.jarsigner` was probably removed between 20 and 21